### PR TITLE
Move config to di instead of di-web

### PR DESCRIPTION
**Pull request by Gerych1984**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | #132

### CODE REVIEW

**Comment by xepozz** [comment]
Moving di-web to di means you want to have it also in console runs. Is it so? What use case do you have?

**Review by samdark — APPROVED** [review]

File: .github/workflows/bc.yml [hunk 1]
```diff
@@ -11,4 +11,4 @@ jobs:
       os: >-
         ['ubuntu-latest']
       php: >-
-        ['8.0']
+        ['8.1', '8.2', '8.3']
```
Comment (vjik) [inline .github/workflows/bc.yml:14]:
```suggestion
        ['8.0']
```

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -28,4 +28,4 @@ jobs:
       os: >-
         ['ubuntu-latest', 'windows-latest']
       php: >-
-        ['8.0', '8.1']
+        ['8.1', '8.2', '8.3']
```
Comment (vjik) [inline .github/workflows/build.yml:31]:
```suggestion
        ['8.0', '8.1', '8.2', '8.3']
```

File: .github/workflows/composer-require-checker.yml [hunk 3]
```diff
@@ -30,4 +30,4 @@ jobs:
       os: >-
         ['ubuntu-latest']
       php: >-
-        ['8.0']
+        ['8.1', '8.2', '8.3']
```
Comment (vjik) [inline .github/workflows/composer-require-checker.yml:33]:
```suggestion
        ['8.0', '8.1', '8.2', '8.3']
```

File: .github/workflows/mutation.yml [hunk 4]
```diff
@@ -26,6 +26,6 @@ jobs:
       os: >-
         ['ubuntu-latest']
       php: >-
-        ['8.1']
+        ['8.1', '8.2', '8.3']
```
Comment (vjik) [inline .github/workflows/mutation.yml:29]:
```suggestion
        ['8.3']
```

File: .github/workflows/static.yml [hunk 5]
```diff
@@ -28,4 +28,4 @@ jobs:
       os: >-
         ['ubuntu-latest']
       php: >-
-        ['8.0', '8.1', '8.2']
+        ['8.1', '8.2', '8.3']
```
Comment (vjik) [inline .github/workflows/static.yml:31]:
```suggestion
        ['8.0', '8.1', '8.2', '8.3']
```

**Review by vjik — CHANGES_REQUESTED** [review]
Looks good 👍
Need line in changelog only

**Comment by Gerych1984** [comment]
> Looks good 👍 Need line in changelog only

Done

**Review by vjik — APPROVED** [review]

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**